packages/babel-core/test/fixtures/transformation/source-maps/arrow-function/source-map-visual.txt

```diff
@@ -0,0 +1,41 @@
+    (1:0) var t         <--     (1:0) var t
+          ^                           ^
+
+    (1:4) var t = x     <--     (1:4) var t = f
+              ^                           ^
+
+    (1:5) ar t = x      <--     (1:5) ar t = fu
+              ^                           ^
+
+    (1:8) t = x =>      <--     (1:8) t = funct
+              ^                           ^
+
+    (1:4) var t = x     <--    (1:17) ion t(x) 
+              ^                           ^
+
+    (1:5) ar t = x      <--    (1:18) on t(x) {
+              ^                           ^
+
+    (1:8) t = x =>      <--    (1:19) n t(x) {
+              ^                           ^
+
+    (1:9)  = x => x     <--    (1:20)  t(x) {
+              ^                           ^
+
+    (1:9)  = x => x     <--     (2:2)   retur
+              ^                         ^
```

Comment:
This PR is just for adding tests and fixes will be in follow-ups, but this is weird. The `r` should either be mapped to `=>`'s `=`, or to the `x` after `=>`.